netcore/NetCore.Session.IntegrationTests/NetCore.Session.IntegrationTests.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+
+    <Platforms>x64</Platforms>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
```

Comment:
Do we really have a direct dependency on this?

Reply:
Looks like this is only needed for running tests in Visual Studio. If you run tests from the command line like the CI will do (i.e., dotnet test) this dependency isn't needed. I'm OK with removing it.